Automattic/jetpack
# push-to-mirrors: Log fetched base rev when preparing push

#### Changes proposed in this Pull Request:

We had an odd situation where a push at [15:54:49](https://github.com/Automattic/jetpack/runs/5467137595?check_suite_focus=true#step:7:5534) seems to not have been
fetched at [15:56:42](https://github.com/Automattic/jetpack/runs/5467157277?check_suite_focus=true#step:7:5539). But it's hard to be sure because the output doesn't
indicate which revision was actually fetched. Let's add that.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Download and unpack the artifact, then try running the script (it won't actually push as long as you don't set `CI` in the environment).
* Merge, then check the output of the push job.
